crates/router/src/connector/bankofamerica/transformers.rs

```diff
@@ -367,6 +445,83 @@ pub struct Avs {
     code_raw: String,
 }
 
+impl
+    TryFrom<(
+        &BankOfAmericaRouterData<&types::PaymentsCompleteAuthorizeRouterData>,
+        payments::Card,
+    )> for BankOfAmericaPaymentsRequest
+{
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        (item, ccard): (
+            &BankOfAmericaRouterData<&types::PaymentsCompleteAuthorizeRouterData>,
+            payments::Card,
+        ),
+    ) -> Result<Self, Self::Error> {
+        let email = item.router_data.request.get_email()?;
+        let bill_to = build_bill_to(item.router_data.get_billing()?, email)?;
```

Comment:
Do we need this to be added in dynamic fields? since this is for 3ds?

Reply:
These fields are required in authorize call (even for 3ds). If we not pass these fields the connectors is throwing error.

<img width="1303" alt="Screenshot 2024-01-16 at 4 56 02 PM" src="https://github.com/juspay/hyperswitch/assets/41580413/1bd96930-9cc0-48ad-ad41-b6bc38610677">